src/config.py

```diff
@@ -17,7 +17,14 @@ def load_config(configfile):
     config['mail_alert_address'] = parser.get('settings', 'mail_alert_address')
     config['email_alert'] = parser.get('settings', 'email_alert')
     config['uptime_threshold'] = parser.get('settings', 'uptime_threshold')
+    config['terminate_long_running'] = parser.get('settings', 'terminate_long_running')
+    if config['terminate_long_running'].lower == "false":
```

Comment:
Yea I was looking for a way to do it more natively with configparser, but haven't found a way.